metabase/metabase
# Fix not being able to use field filters in databases without schema

Fixes https://github.com/metabase/metabase/issues/29786
Reproduces https://github.com/metabase/metabase/issues/29786

In 0.45 it worked because the `ErrorBoundary` simply ignored any exceptions during rendering and now it shows a dedicated page.

How to verify:
- New -> SQL Query
- `SELECT * FROM PRODUCTS WHERE {{f1}} AND {{f2}}`
- Make `f1` and `f2` field filters, set their values, and run the query
- You should be able to run the query and get results

<img width="1086" alt="Screenshot 2023-04-12 at 15 13 08" src="https://user-images.githubusercontent.com/8542534/231453695-c36f97f4-99f4-4fc4-b8ee-dbdf6cbf1be8.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30029)
